wowup-electron/src/assets/i18n/de.json

```diff
@@ -217,19 +217,19 @@
         "GAME_VERSION_COLUMN_HEADER": "Spielversion",
         "LATEST_VERSION_COLUMN_HEADER": "Aktuelle Version",
         "PROVIDER_COLUMN_HEADER": "Anbieter",
-        "PROVIDER_RELEASE_CHANNEL": "Anbieter Kanal",
-        "RELEASED_AT_COLUMN_HEADER": "Veröffentlicht am",
+        "PROVIDER_RELEASE_CHANNEL": "Anbieter-Channel",
```

Comment:
Anbieter-Kanal, bleiben wir deutsch

Reply:
Ich denke hier ist das in diesem Kontext OK "Channel" zu nehmen, heutzutage und im "Internetkontext" sagt kaum einer Kanal - auf Twitch würde man eher "Channel" sagen, zB. Kann ich aber gerne ändern, wäre auch gut mehr Meinungen dazu zu haben!